lib/src/ui/map.dart

```diff
@@ -1409,3 +1419,15 @@ class IControl extends JsObjectWrapper<IControlJsImpl> {
   /// Creates a new IControl from a [jsObject].
   IControl.fromJsObject(IControlJsImpl jsObject) : super.fromJsObject(jsObject);
 }
+
+class MapboxStyle extends JsObjectWrapper<MapboxStyleJsImpl> {
+  MapboxStyle.fromJsObject(MapboxStyleJsImpl jsObject)
+      : super.fromJsObject(jsObject);
+
+  List<dynamic> get layers => jsObject.layers;
+}
+
+@JS('Style')
+class MapboxStyleJsImpl {
+  external List<dynamic> layers;
+}
```

Comment:
Once we have decided what the are going to do, this will be deleted or it will be moved into its own file.